src/sentry/testutils/cases.py

```diff
@@ -3,7 +3,7 @@
 import responses
 import sentry_kafka_schemas
 
-from sentry.sentry_metrics.use_case_id_registry import REVERSE_METRIC_PATH_MAPPING, UseCaseID
```

Comment:
Can `REVERSE_METRIC_PATH_MAPPING` be removed completely? According to [this comment](https://github.com/getsentry/sentry/blob/bf993a1e5dc82216995138a97033ba4cc973333e/src/sentry/sentry_metrics/use_case_id_registry.py#L25) it was always meant to be a temporary mapping.

Reply:
Not at the moment (unfortunately), there is still some places that I didn't touch (mostly because I don't quite know what they do) where we still use UseCaseKey and we convert to and from UseCaseID.

We'll have to have another go to remove it completely (maybe we could ask for help from the people that wrote slicing_rooter for example).